src/components/carousel/README.md

```diff
@@ -11,6 +11,9 @@ controls and indicators. Please be aware that nested carousels are **not** suppo
                 style="text-shadow: 1px 1px 2px #333;"
                 controls
                 indicators
+                focus
+                wrap
+                pause="hover"
```

Comment:
why is this a string, when the only option is really  on or off (true or false)? And it is either on hover or not.